graphics/macgui/macfontmanager.cpp

```diff
@@ -327,12 +331,10 @@ const char *MacFontManager::getFontName(int id, int size, int slant, bool tryGen
 		slant = 0;
 	}
 
-	snprintf(name, 128, "%s-%d-%d", n.c_str(), slant, size);
-
-	return name;
+	return Common::String::format("%s-%d-%d", n.c_str(), slant, size);
```

Comment:
A Common::String variable is still needed here, as Common::String::format() is a static method, so no new instance of a Common::String is created. You could do:

    return Common::String(Common::String::format(...));

Reply:
This code seems fine to me since Common::String::format() returns a Common::String() already.

Reply:
True :)